pkg/bugzilla/bugzilla.go

```diff
@@ -110,10 +110,11 @@ func (c *Verifier) VerifyBugs(bugs []int) []error {
 			comments, err := c.bzClient.GetComments(bugID)
 			if err != nil {
 				errs = append(errs, fmt.Errorf("Failed to get comments on bug %d: %v", bug.ID, err))
+				continue
 			}
 			var alreadyCommented bool
 			for _, comment := range comments {
-				if comment.Text == message {
+				if comment.Text == message && comment.Creator == "openshift-bugzilla-robot" {
```

Comment:
Can you send an API request to Bugzilla and see what your identity is, so that changing the token/account we use does not cause this to hard-fail?

Reply:
I wasn't able to find any API calls that could tell me the current identity of the caller's token